docs/requirements.txt

```diff
@@ -1,6 +1,7 @@
 furo == 2024.8.6
+setuptools == 75.6.0
```

Comment:
https://github.com/flashinfer-ai/flashinfer/pull/666 https://github.com/flashinfer-ai/flashinfer/pull/668

Thank you for submitting these pull requests. I've been reviewing them and wanted to share some observations and concerns:

At present, the changes appear to be causing our nightly build to fail, and I'm having trouble identifying the intended benefits that would offset this disruption.
Regarding the version number changes - while I appreciate the thought behind including the git hash and date by default, I'm wondering if we might want to reconsider this approach, as it doesn't align with PEP 440 specifications.

Would you be open to discussing the goals you're trying to achieve with these changes? This could help us find a solution that meets your objectives while maintaining build stability and compatibility.

Reply:
Those PRs focus on moving torwards a more generalized, stable and widely used method to produce packages, which can avoid unfixable breakages in the future. Breakages introduced by themselves are annoying enough though...

Reply:
Hi @zhyncs I think these steps are necessary and our packaging infrastructures are indeed outdated.

I would say it's my fault instead of contributors that I haven't set up CI because of limited resources, but the situation is changing soon :)

Reply:
Exactly. Make FlashInfer better!